File: api/src/main/java/com/hirrao/javaee/entity/DrugAlert.java
package com.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.sql.Date;
import java.sql.Time;

@Getter
@Setter
@AllArgsConstructor
@TableName("DrugAlert")
public class DrugAlert {
    @TableId(value = "alertId")
    //@TableField("alertId")
    private Long alertId;
    private Long uid;
    @TableField("drugId")
    private Long drugId;
    @TableField("alertTime")
    private Time alertTime;
    @TableField("eatTime")
    private Date eatTime;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/DrugAlert.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import java.sql.Date
import java.sql.Time

@TableName("DrugAlert")
data class DrugAlert(
    @TableId(value = "alertId") //@TableField("alertId")
    val alertId: Long? = null,
    val uid: Long? = null,

    @TableField("drugId") val drugId: Long? = null,

    @TableField("alertTime") val alertTime: Time? = null,

    @TableField("eatTime") val eatTime: Date? = null,
)